test/models/track_test.rb

```diff
@@ -29,23 +29,13 @@ class TrackTest < ActiveSupport::TestCase
     assert_equal track.slug, track.to_param
   end
 
-  test "Retrieves test_regexp for csharp" do
-    track = create :track, slug: 'csharp'
-    assert_equal(/.+test[.]rb$/, track.test_regexp)
-  end
-
-  test "Retrieves test_regexp for ruby" do
-    track = create :track, slug: 'ruby'
+  test "Retrieves test_regexp" do
+    track = create :track
     assert_equal(/test/, track.test_regexp)
   end
 
-  test "Retrieves ignore_regexp for csharp" do
-    track = create :track, slug: 'csharp'
-    assert_equal(/[iI]gnore/, track.ignore_regexp)
-  end
-
-  test "Retrieves ignore_regexp for ruby" do
-    track = create :track, slug: 'ruby'
+  test "Retrieves ignore_regexp" do
+    track = create :track
```

Comment:
These two tests checked both when the regexp was specified in a track, and when it wasn't (ie checked the default value). So we need to keep them both. I believe the two repos (`track-naked` and `track-with-exercises`) had these different characteristics.

Reply:
Ah, there was another test that already tested that (I think it was the Git::Track class), but I can add it here too.

Reply:
If it's already tested elsewhere, then that's fine. As long as it has proper coverage somewhere, I'm ok with that :)